lib/perl/Genome/Model/Tools/Vcflib.pm

```diff
@@ -0,0 +1,182 @@
+package Genome::Model::Tools::Vcflib;
+
+use strict;
+use warnings;
+
+use Genome;
+
+my $DEFAULT = '1.0';
+
+class Genome::Model::Tools::Vcflib {
+    is  => 'Command::V2',
+    has => [
+        use_version => {
+            is            => 'Version',
+            is_optional   => 1,
+            default_value => $DEFAULT,
+            doc           => "Version of tophat to use, default is $DEFAULT"
```

Comment:
This is not `tophat`.  It is also not necessary to list the default in the `doc` line as the `default_value` is propagated into the `--help` text already.